vars/main.yml

```diff
@@ -431,6 +431,35 @@ pgbackrest_cron: []
 #  - { name: "Full Backup", type: "full", hour: "06", minute: "30", dow: "0" }
 #  - { name: "Diff Backup", type: "diff", hour: "06", minute: "30", dow: "1-6" }
 
+cron_jobs: []
+#  - name: "PgBackerest: Full Backup"
+#    file: /etc/cron.d/pgbackrest
+#    user: "{{ patroni_superuser_username }}"
+#    minute: "30"
+#    hour: "6"
+#    day: "*"
+#    month: "*"
+#    weekday: "0"
+#    job: "/usr/bin/pgbackrest --type=full --stanza={{ pgbackrest_stanza }} backup"
+#  - name: "PgBackerest: Diff Backup"
+#    file: /etc/cron.d/pgbackrest
+#    user: "{{ patroni_superuser_username }}"
+#    minute: "30"
+#    hour: "6"
+#    day: "*"
+#    month: "*"
+#    weekday: "1-6"
+#    job: "/usr/bin/pgbackrest --type=diff --stanza={{ pgbackrest_stanza }} backup"
+# Or for walg
+#  - name: "WAL-G: Create daily backup"
+#    user: "{{ patroni_superuser_username }}"
+#    minute: "30"
+#    hour: "6"
+#    day: "*"
+#    month: "*"
+#    weekday: "*"
+#    job: "[ '200' = $(curl -s -o /dev/null -w '\\%{http_code}' http://{{ hostvars[inventory_hostname]['inventory_hostname'] }}:8008) ] && /usr/local/bin/wal-g backup-push"
```

Comment:
```suggestion
#    job: "if [ "$(psql -tAXc 'select pg_is_in_recovery()')" = "f" ]; then wal-g backup-push; fi"
```
run the WAL-G backup process on the Primary server.

Reply:
This command allows you to determine the master according to patroni
> [ '200' = $(curl -s -o /dev/null -w '\\%{http_code}' http://{{ hostvars[inventory_hostname]['inventory_hostname'] }}:8008) ]

Example for bash
`curl -s -o /dev/null -w '%{http_code}' http://10.48.5.196:8008`

Reply:
I usually prefer to call the `pg_is_in_recovery()` function, and I think this method is more readable.

Reply:
I think we can leave your example as well